.github/workflows/github-actions.yml

```diff
@@ -10,29 +11,75 @@ jobs:
             AWS_DEFAULT_OUTPUT: text
             THERAPY_NORM_DB_URL: http://localhost:8001
             TEST: THERAPY_TEST
+        strategy:
+            matrix:
+              python-version: [3.8, 3.9]
         steps:
             - uses: actions/checkout@v2
 
             - name: Setup Python
               uses: actions/setup-python@v1
               with:
-                  python-version: 3.8
+                  python-version: ${{ matrix.python-version }}
 
             - name: Install dependencies
               run: |
-                  python3 -m pip install pipenv
-                  pipenv install --dev
+                  python3 -m pip install -e ".[test]"
 
             - name: Build local DynamoDB
               run: |
                   chmod +x ./tests/scripts/dynamodb_build.bash
                   ./tests/scripts/dynamodb_build.bash
 
+            - name: Run tests
+              run: python3 -m tox -e githubactions
+    lint:
```

Comment:
Can we merge lint and typing into one job with both steps? Seems a little silly to break them into two separate jobs